basin3d/plugins/usgs.py

```diff
@@ -636,8 +645,8 @@ def list(self, **kwargs):
                     # TODO: write some tests for this which will require mocking a data return.
                     # Only filter if quality_checked is True
                     # if QUERY_PARAM_RESULT_QUALITY not in kwargs or not kwargs[QUERY_PARAM_RESULT_QUALITY] or \
-                            # (QUERY_PARAM_RESULT_QUALITY in kwargs and kwargs[
-                                # QUERY_PARAM_RESULT_QUALITY] == result_quality):
+                    #         (QUERY_PARAM_RESULT_QUALITY in kwargs and kwargs[
+                    #          QUERY_PARAM_RESULT_QUALITY] == result_quality):
```

Comment:
Let's remove this commented out code and write an issue ticket for it.

Reply:
Any chance we can leave it for now? The next task is reworking result_quality so I would deal with this comment in the next PR. Would be easier for me to think about / address then. I'm not totally sure what needs to happen here.

Reply:
ok.